hack/jenkins/e2e.sh

```diff
@@ -472,6 +472,21 @@ case ${JOB_NAME} in
     : ${ENABLE_DEPLOYMENTS:=true}
     ;;
 
+  kubernetes-e2e-gke-subnet)
+    : ${DOGFOOD_GCLOUD:="true"}
+    : ${E2E_CLUSTER_NAME:="jkns-gke-subnet"}
+    : ${E2E_NETWORK:="auto-subnet"}
```

Comment:
This network is manually created with custom settings. Should add a comment here stating how to recreate it (ideally with gcloud command to use) if it gets deleted accidentally.